py3status/modules/net_rate.py

```diff
@@ -74,12 +76,12 @@ def currentSpeed(self):
         if not isinstance(self.interfaces_blacklist, list):
             self.interfaces_blacklist = self.interfaces_blacklist.split(',')
 
-        # == 6 characters (from MULTIPLIER_TOP + dot + self.precision)
+        # == 6 characters (from self.multiplier_top + dot + self.precision)
         if self.precision > 0:
-            self.left_align = len(str(MULTIPLIER_TOP)) + 1 + self.precision
+            self.left_align = len(str(self.multiplier_top)) + 1 + self.precision
         else:
-            self.left_align = len(str(MULTIPLIER_TOP))
-        self.value_format = "{value:%s.%sf} {unit}" % (
+            self.left_align = len(str(self.multiplier_top))
+        self.value_format = "{value:%s.%sf} {unit:>4s}" % (
```

Comment:
I think that this should be made available to the user as `format_total = '{value} {unit}'` that way they can change it as the want - you can set a sensible default.

Look at the rate_counter modules `format_money` as an example